Resources/Locale/en-US/job/job-names.ftl

```diff
@@ -49,6 +49,7 @@ job-name-visitor = Visitor
 
 # Role timers - Make these alphabetical or I cut you
 JobAtmosphericTechnician = Atmospheric Technician
+JobAdministrativeAssistant = Administrative Assistant
```

Comment:
This needs to be in a Delta file or have a Delta tag; you can't add tags in .ftl but the game doesn't shit bricks if you add one though.

Reply:
Moved to Delta V's job-names.ftl